qemu/tests/uefi_check_secure_MOR.py

```diff
@@ -0,0 +1,92 @@
+    """
+    Please make sure the guest installed with signed driver
+    Verify Secure MOR control feature using Device Guard tool in Windows guest:
+
+    1) Boot up a guest.
+    2) Check if Secure Boot is enable.
+    3) Download Device Guard and copy to guest.
+    4) Enable Device Guard and check the output.
+    5) Reboot guest.
+    5) Run Device Guard and check the output.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def execute_powershell_command(command):
+        status, output = session.cmd_status_output(command)
+        logging.info("command:%s" % command)
+        logging.info("command output:%s" % output)
+        if status != 0:
+            test.fail("execute command fail: %s" % output)
+        return output
+    timeout = int(params.get("timeout", 360))
+    params['ovmf_vars_filename'] = 'OVMF_VARS.secboot.fd'
+    params['start_vm'] = 'yes'
+    params['cpu_model_flags'] = ',hv-passthrough'
+    env_process.preprocess_vm(test, params, env, params['main_vm'])
+    vm = env.get_vm(params["main_vm"])
+    session = vm.wait_for_serial_login(timeout=timeout)
+
+    check_cmd = params['check_secure_boot_enabled_cmd']
+    dgreadiness_tool_path = params['dgreadiness_tool_path']
+    executionPolicy_command = params['executionPolicy_command']
+    enable_command = params['enable_command']
+    ready_command = params['ready_command']
+
+    try:
+        status, output = session.cmd_status_output(check_cmd)
+        if status != 0:
+            test.fail('Secure boot is not enabled. The actual output is %s'
+                      % output)
+
+        # Download Device Guard to host, and copy to guest
+        dgreadiness_tool = params["dgreadiness_tool"]
+        src_path = params['src_path']
+        download_cmd = params["download_cmd"]
+        dst_path = params["dst_path"]
+        unzip_command = params["unzip_command"]
+        logging.info("Download Device Guuard and copy it to guest.")
+        process.system(download_cmd, timeout=360)
+        process.system(unzip_command, timeout=360)
+        if not os.path.exists(dgreadiness_tool):
+            test.error("Device Guard is not exist, maybe it is not "
+                       "successfully downloaded ")
+        s, o = session.cmd_status_output("mkdir %s" % dst_path)
+        if s and "already exists" not in o:
+            test.error("Could not create Device Guard directory in "
+                       "VM '%s', detail: '%s'" % (vm.name, o))
+        logging.info("Copy Device Guard to guest")
+        vm.copy_files_to(src_path, dst_path)
+        os.remove(dgreadiness_tool)
+
+        execute_powershell_command(dgreadiness_tool_path)
+        execute_powershell_command(executionPolicy_command)
+        output = execute_powershell_command(enable_command)
+        check_enable_info = params['check_enable_info']
+        if output.find(check_enable_info) == -1:
```

Comment:
how about if check_enable_info in output? simple and effective,  if there could be multiple keywords(key1|key2|..) checking, use re.search might be better